packages/core/package.json

```diff
@@ -17,7 +17,7 @@
     "express": "^4.15.3",
     "file-icons-js": "^1.0.3",
     "font-awesome": "^4.7.0",
-    "humane-js": "^3.2.2",
+    "fs-extra": "^4.0.2",
```

Comment:
why do you need it?

Reply:
@akosyakov, I think merge conflict resolution failed.